README.md

```diff
@@ -136,6 +136,20 @@ $ station logs --follow
 ...
 ```
 
+### `$ station --listen`
+
+Open HTTP API.
```

Comment:
Maybe instead of `--listen` argument, we can implement the following flag:
- If `PORT` is set to some value (including `0`), we start the HTTP server
- If `PORT` is not specified, we don't create any server.

It's less explicit than `--listen`, so maybe it's not a good idea. Posting it just for consideration.

Reply:
Good point, it's not a command it's just an option, meaning that it will run the basic `$ station` command, but also open an HTTP server.

> IMO, --listen should be an optional argument for the default command (station) that runs the main station process & modules.

That's exactly how it is, I'm going to improve the README.

> It's less explicit than --listen, so maybe it's not a good idea. Posting it just for consideration.

Those are my conerns as well. And we do read `$PORT` already, but only if `--listen` is set.